<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="pageMultiWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="SRStdMultiWidget" name="multiWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="frameLegacy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {
  background: rgb(0, 0, 0)
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="toolButtonDefault">
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonShow">
             <property name="text">
              <string>Show</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonView">
             <property name="text">
              <string>View</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonTools">
             <property name="text">
              <string>Tools</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonAdd">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="popupMode">
              <enum>QToolButton::InstantPopup</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonSelect">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonMarking">
             <property name="text">
              <string>Marking</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonEdit">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonDelete">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonUndo">
             <property name="text">
              <string>Undo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonRedo">
             <property name="text">
              <string>Redo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonLoad">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonExport">
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonAlign">
             <property name="text">
              <string>Align</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonPlanning">
             <property name="text">
              <string>Planning</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonFusion">
             <property name="text">
              <string>Fusion</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonIntraOp">
             <property name="text">
              <string>IntraOp</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QToolButton" name="toolButton_7">
           <property name="text">
            <string>Screw View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_9">
           <property name="text">
            <string>Path View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_8">
           <property name="text">
            <string>Planar View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>Cambered View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_15">
           <property name="text">
            <string>Cylinder View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_14">
           <property name="text">
            <string>Wedge View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_12">
           <property name="text">
            <string>Hemivertebra View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_11">
           <property name="text">
            <string>cobb</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>Screw</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="text">
          <string>Plate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_5">
         <property name="text">
          <string>Interval</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_6">
         <property name="text">
          <string>Lateral</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_10">
         <property name="text">
          <string>Posterior</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_13">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="dummy"/>
        <widget class="SegmentationWidget" name="segmentationWidget"/>
        <widget class="PlanningWidget" name="planningWidget"/>
        <widget class="ToolsWidget" name="toolsWidget"/>
        <widget class="CasePlanningWidget" name="casePlanningWidget"/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="toolButtonPrevious">
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonNext">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_Whole_Spine_CT">
   <property name="text">
    <string>&amp;Whole Spine CT</string>
   </property>
  </action>
  <action name="action_Single_Vertebra_CT">
   <property name="text">
    <string>&amp;Single Vertebra CT</string>
   </property>
   <property name="toolTip">
    <string>Single Vertebra CT</string>
   </property>
  </action>
  <action name="action_Whole_Spine_X_Ray">
   <property name="text">
    <string>&amp;Whole Spine X Ray</string>
   </property>
  </action>
  <action name="action_Whole_Spine_MRI">
   <property name="text">
    <string>&amp;Whole Spine MRI</string>
   </property>
  </action>
  <action name="action_Single_Vertebra_MRI">
   <property name="text">
    <string>&amp;Single Vertebra MRI</string>
   </property>
  </action>
  <action name="action_Segmentation">
   <property name="text">
    <string>&amp;Segmentation</string>
   </property>
  </action>
  <action name="action_Planning">
   <property name="text">
    <string>&amp;Planning</string>
   </property>
  </action>
  <action name="action_Length">
   <property name="text">
    <string>&amp;Length</string>
   </property>
  </action>
  <action name="action_Angle">
   <property name="text">
    <string>&amp;Angle</string>
   </property>
  </action>
  <action name="action_Screw_Planning">
   <property name="text">
    <string>&amp;Screw Planning</string>
   </property>
  </action>
  <action name="action_Path_Planning">
   <property name="text">
    <string>&amp;Path Planning</string>
   </property>
  </action>
  <action name="action_Plate_Planning">
   <property name="text">
    <string>&amp;Plate Planning</string>
   </property>
  </action>
  <action name="action_Interval_Planning">
   <property name="text">
    <string>&amp;Interval Planning</string>
   </property>
  </action>
  <action name="action_Lateral_Planning">
   <property name="text">
    <string>&amp;Lateral Planning</string>
   </property>
  </action>
  <action name="action_Posterior_Planning">
   <property name="text">
    <string>&amp;Posterior Planning</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SRStdMultiWidget</class>
   <extends>QWidget</extends>
   <header>SRStdMultiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SegmentationWidget</class>
   <extends>QWidget</extends>
   <header>SegmentationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlanningWidget</class>
   <extends>QWidget</extends>
   <header>PlanningWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolsWidget</class>
   <extends>QWidget</extends>
   <header>ToolsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CasePlanningWidget</class>
   <extends>QWidget</extends>
   <header>CasePlanningWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
